build/circle-test.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -e
+
+outdir="${TMPDIR}"
+if [ "${CIRCLE_ARTIFACTS}" != "" ]; then
+    outdir="${CIRCLE_ARTIFACTS}"
+fi
+
+builder=$(dirname $0)/builder.sh
+
+# 1. Run "make check" to verify coding guidelines. 
+echo "make check"
+time ${builder} make GITHOOKS= check | tee "${outdir}/check.log"
```

Comment:
Done. It's necessary to prevent bootstrapping. If the `GITHOOK` files don't exist, bootstrapping is preformed.